cli/graph_util.rs

```diff
@@ -208,56 +206,34 @@ pub struct CreateGraphOptions<'a> {
   pub graph_kind: GraphKind,
   pub roots: Vec<ModuleSpecifier>,
   pub is_dynamic: bool,
-  /// Whether to do fast check on workspace members. This is mostly only
-  /// useful when publishing.
-  pub workspace_fast_check: bool,
   /// Specify `None` to use the default CLI loader.
   pub loader: Option<&'a mut dyn Loader>,
 }
 
-pub struct ModuleGraphBuilder {
+pub struct ModuleGraphCreator {
```

Comment:
This was broken up into the `ModuleGraphCreator`, which creates module graphs, then the `ModuleGraphBuilder`, which builds onto existing module graphs. The builder is used at a lower level.